v3/newrelic/segments.go

```diff
@@ -286,6 +286,7 @@ func StartExternalSegment(txn *Transaction, request *http.Request) *ExternalSegm
 	s := &ExternalSegment{
 		StartTime: txn.StartSegmentNow(),
 		Request:   request,
+		Host:      request.Host,
```

Comment:
With our experimentation, this doesn't actually reflect the value set in the "Host" header. Host just gets set to whatever was actually requested in the URL -- in your case, it would be "localhost." Perhaps you're expecting the request object to pull the header automatically?

Reply:
@eightnoteight Can you confirm if @RichVanderwal's suggestion adequately solves your need without changing the existing behavior of the SDK, or if it really is a better solution to make the proposed change? Thanks!